### ISSUE TITLE
Add realtime HREF to ensemble verification

### ISSUE THREAD

**Issue by JeffHamiltonNOAA** [NOAA-GSL/VxLegacyIngest#475]
Restart the HREF ensemble verification for precip and reflectivity. This ran a year ago with the old system (before proper neighborhood techniques were added), so this will take some adjustment to get it up and running

**Comment by bonnystrong** [comment]
Depends on the new ensemble workflow running.

**Comment by bonnystrong** [comment]
Depends on #487 being completed.

**Comment by bonnystrong** [comment]
No longer relevant.  Up to Trevor if he wants to add it.
